gm4_soul_glass/data/gm4_soul_glass/functions/furnace/modify_output.mcfunction

```diff
@@ -1,5 +1,5 @@
 #@s = none
 #run from finish_smelt
 
-execute at @e[tag=gm4_sg_smelted] run data modify block ~ ~ ~ Items[{Slot:2b}].tag set value {gm4_soul_glass:1b,Enchantments:[{}],display:{Name:'{"text":"Soul Glass","italic":false}'}}
+execute at @e[tag=gm4_sg_smelted] run data modify block ~ ~ ~ Items[{Slot:2b}].tag set value {CustomModelData:2,gm4_soul_glass:1b,Enchantments:[{}],display:{Name:'{"text":"Soul Glass","italic":false}'}}
```

Comment:
This CMD should also be added to the `destroy.mcfunction` in the item summon command as well.